### PR TITLE
Unexpected PHP behavior

### PR DESCRIPTION
**Pull request by sylfabre**
Following up on symfony/symfony#28259

### CODE REVIEW

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -473,6 +473,13 @@ namespaces in the ``phpunit.xml`` file, as done for example in the
             </listener>
         </listeners>
     </phpunit>
+    
+Under the hood, this component defines time and dns functions in the tested
+class namespace when the annotated test is run. Because of a PHP unexpected
+behavior (see https://bugs.php.net/bug.php?id=64346), a test may pass when
+run alone but fail when run in a tests suite if the tested class is used
+before these time and dns functions are defined. Configuring the mocked
```
[the remark this answers is not in the document]
Comment (sylfabre) [reply components/phpunit_bridge.rst]: @javiereguiluz I've added the example provided by nicolas-grekas on https://github.com/symfony/symfony/issues/28259 

Do you think it's enough?

**Comment by sylfabre** [comment]
@nicolas-grekas done

**Review by ndench — APPROVED** [review]
This looks good to me, thanks!

File: components/phpunit_bridge.rst [hunk 2]
```diff
@@ -473,6 +473,27 @@ namespaces in the ``phpunit.xml`` file, as done for example in the
...
     </phpunit>
+    
+Under the hood, a PHPUnit listener injects the mocked functions in the tested
+classes' namespace. In order to work as expected, the listener has to run before
+the tested class ever runs. By default, the mocked functions are created when the
+annotation are found and the corresponding tests are run. Depending on how your
+tests are constructed, this might be too late. In this case, you will need to declare
+the namespaces of the tested classes in your phpunit.xml.dist
+
+````
```
Comment (kunicmarko20) [inline components/phpunit_bridge.rst]: `.. code-block:: xml` should probably be used here instead of 4 backticks

Comment (sylfabre) [reply components/phpunit_bridge.rst]: fixed!

**Pull request closed without merging**